LIBRARY.md

```diff
@@ -55,16 +63,14 @@ make
 sudo make install
 
 # To install the python module
-sudo make python-install
+sudo make jrl-python-install
```

Comment:
Would it be worthwhile to include how to use `JRL_BUILD_PYTHON` in the instructions down here? Since `-DJRL_BUILD_PYTHON=ON` might not be so obvious to those not familiar with cmake